docs/learn/learn-staking-miner.md

```diff
@@ -0,0 +1,79 @@
+---
+id: learn-staking-miner
+title: Staking Miner
+sidebar_label: Staking Miner
+description: Learn about Phragmén optimization on-chain via staking miners
+keywords: [staking, npos, miner, phragmén]
+slug: ../learn-staking-miner
+---
+
+:::caution
+The staking-miner code is experimental and it is still in development phase. Use is at your own discression, some funds might be lost.
```

Comment:
Typo - discretion. Also, leave an empty space above and below the content of a docusaurus admonition as we might run prettier on the doc https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier

```suggestion

The staking-miner code is experimental and it is still in development phase. Use is at your own discretion, as there is a risk of losing some funds.

```